nodeJS/express_and_mongoose/deployment.md

```diff
@@ -100,8 +100,15 @@ Whatever your circumstances, we've got you covered. Here are the PaaS providers
 - $20 a month should be enough to host eight apps (including three apps for free).
 - Fly.io charges $0.15/GB of RootFS for machines stopped for 30 days.
 
-##### Fly.io: Free Plan
+##### Fly.io: Hobby Plan
 
+- New customers get a one-time $5 free trial credit to test Fly.io at no cost. After the credit has been used, you will be be automatically placed on the $5/month Hobby plan subscription
```

Comment:
I think it's worth specifying this credit is only used if you exceed the free resource allowance. E.g.
> The credit will only start being used if you exceed the free resource allowance, and once it's been used up, you will be automatically placed on the $5/month Hobby plan.

(And the same of course for the other lesson file)